app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,35 @@
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
+
+/* universal */
+body {
+padding-top: 100px;
+}
+section {
+overflow: auto;
+}
+textarea {
+resize: vertical;
+}
+.center {
+text-align: center;
+}
+.center h1 {
+margin-bottom: 100px;
+}
+
+/* header */
+#logo {
+float: left;
+margin-right: 10px;
+font-size: 1.7em;
+color: #fff;
+text-transform: uppercase;
+letter-spacing: -1px;
+padding-top: 9px;
+font-weight: bold;
+}
+#logo:hover {
+color: #fff;
+text-decoration: none;
```

Comment:
add indent 2 spaces for this file, move color variables to other class, then call back here. Add end line for this file